app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,26 @@
+class RentsController < ApiController
+  before_action :authenticate_user!
+
+  include Wor::Paginate
+
+  def create
+    @rent = Rent.new(rent_create_params)
+    if !@rent.valid?
```

Comment:
hmm why not @rent.valid? and swapping the conditions? I think it's easier to read.

Reply:
The `.save` method returns true or false. This return value depends on the validation of the model. So if a `@rent.save` returns true, it means that it saved it in the database because all validations have passed.

What I'm trying to say here is that you can use directly .save instead of first checking if the model is valid